docs/installation/configuring-the-github-secrets.md

```diff
@@ -1,8 +1,8 @@
 # Configuring the Github secrets
 
 1. Get Dockerhub token
-   ![Get Dockerhub token](images/dockerhub.PNG)
+   ![Get Dockerhub token](/docs/images/dockerhub.PNG)
```

Comment:
That url is broken https://breatheco-de.github.io/docs/images/dockerhub.PNG I will try with `../../images/dockerhub.PNG`

Reply:
Fixed https://breatheco-de.github.io/apiv2/installation/configuring-the-github-secrets/ 👍